bindings/node/etc/build-static.sh

```diff
@@ -23,22 +23,30 @@ pushd $BUILD_DIR #./deps/tmp
 pushd $TOP_DIR
 # build and install bson
 
-# NOTE: we are setting -DCMAKE_INSTALL_LIBDIR=lib to ensure that the built 
+# NOTE: we are setting -DCMAKE_INSTALL_LIBDIR=lib to ensure that the built
 # files are always installed to lib instead of alternate directories like
 # lib64.
 # NOTE: On OSX, -DCMAKE_OSX_DEPLOYMENT_TARGET can be set to an OSX version
 # to suppress build warnings. However, doing that tends to break some
 # of the versions that can be built
-export BSON_EXTRA_CMAKE_FLAGS="-DCMAKE_INSTALL_LIBDIR=lib -DCMAKE_OSX_DEPLOYMENT_TARGET=\"10.12\""
-
+export BSON_EXTRA_CMAKE_FLAGS="-DCMAKE_INSTALL_LIBDIR=lib -DCMAKE_OSX_DEPLOYMENT_TARGET=\"10.12\" -DCMAKE_C_FLAGS_RELWITHDEBINFO=\"/MT\""
 . ${TOP_DIR}/libmongocrypt/.evergreen/build_install_bson.sh
 
 popd #./deps/tmp
 
 # build and install libmongocrypt
 pushd libmongocrypt-build #./deps/tmp/libmongocrypt-build
-$CMAKE -DDISABLE_NATIVE_CRYPTO=1 -DCMAKE_C_FLAGS="-fPIC" -DCMAKE_PREFIX_PATH=$DEPS_PREFIX -DCMAKE_INSTALL_PREFIX=$DEPS_PREFIX  -DCMAKE_INSTALL_LIBDIR=lib -DCMAKE_OSX_DEPLOYMENT_TARGET="10.12" $LIBMONGOCRYPT_DIR
-  make -j8 install
+
+CMAKE_FLAGS="-DDISABLE_NATIVE_CRYPTO=1 -DCMAKE_C_FLAGS=\"-fPIC\" -DCMAKE_INSTALL_LIBDIR=lib "
+if [ "$OS" == "Windows_NT" ]; then
+  WINDOWS_CMAKE_FLAGS="-Thost=x64 -A x64 -DCMAKE_C_FLAGS_RELWITHDEBINFO=\"/MT\""
+  $CMAKE $CMAKE_FLAGS $WINDOWS_CMAKE_FLAGS -DCMAKE_PREFIX_PATH=$DEPS_PREFIX -DCMAKE_INSTALL_PREFIX=$DEPS_PREFIX "`cygpath -w $LIBMONGOCRYPT_DIR`"
+else
+  $CMAKE $CMAKE_FLAGS -DCMAKE_PREFIX_PATH=$DEPS_PREFIX -DCMAKE_INSTALL_PREFIX=$DEPS_PREFIX -DCMAKE_OSX_DEPLOYMENT_TARGET="10.12" $LIBMONGOCRYPT_DIR
+fi
+
+$CMAKE --build . --target install --config RelWithDebInfo
```

Comment:
is this analogous to the `make -j8 install`?

Reply:
this is a portable version of that method, works on windows and posix systems.